tapgarden/custodian.go

```diff
@@ -579,6 +579,28 @@ func (c *Custodian) mapToTapAddr(walletTx *lndclient.Transaction,
 		return nil, fmt.Errorf("unable to encode address: %w", err)
 	}
 
+	// Skip already completed events.
```

Comment:
So this'll suppress duplicate notification events?

Reply:
Yes. Otherwise if we have a pending event at block X we would get notifications for all events between block X and now (even if they are already complete).